portfolio/src/main/webapp/comments.js

```diff
@@ -69,59 +68,78 @@ function addMessage(pageNumber = 0, sortMethod = {"feature": "timestamp",
   });
 }
 
+
 /**
  * Build the elements of an individual message using JSON data
  * Also create a delete button
  */
 function createMessageElements(comment) {
```

Comment:
this method is really long and becomes confusing to read with so many similarly-named elements.  break it up into smaller methods for each element.